presto-main/src/main/java/com/facebook/presto/metadata/ViewDefinition.java

```diff
@@ -31,20 +32,24 @@
     private final Optional<String> schema;
     private final List<ViewColumn> columns;
     private final Optional<String> owner;
+    private final boolean runAsInvoker;
 
     @JsonCreator
     public ViewDefinition(
             @JsonProperty("originalSql") String originalSql,
             @JsonProperty("catalog") Optional<String> catalog,
             @JsonProperty("schema") Optional<String> schema,
             @JsonProperty("columns") List<ViewColumn> columns,
-            @JsonProperty("owner") Optional<String> owner)
+            @JsonProperty("owner") Optional<String> owner,
+            @JsonProperty("runAsInvoker") boolean runAsInvoker)
```

Comment:
Do i understand correctly that existing view definitions will be parsed correctly because Jackson defaults missing boolean properties to `false`?

Reply:
Am i right that `ViewDefinition` is what gets persisted in a Metastore?
if so, we should have tests like:
```
json literal → deserialization → check parsed correctly
```
to ensure backwards compat with view definitions stored by older Presto vesions

Reply:
Yes, existing views will parse correctly. Good point -- I will add a test for existing views.